src/ensembl/src/shared/components/instant-download/instant-download-protein/InstantDownloadProtein.tsx

```diff
@@ -36,6 +38,15 @@ export type ProteinOption = keyof Partial<ProteinOptions>;
 
 export const proteinOptionsOrder: ProteinOption[] = ['proteinSequence', 'cds'];
 
+export type OnDownloadPayload = {
```

Comment:
Also are we not interested in genomeId?

Reply:
I just thought that it will be the easiest to report back to the parent everything that InstantDownloadProtein knows about. If the parent doesn't care about the genome id, it can just ignore it.

Reply:
I asked in the context of reporting genomeId - whether it helps in analytics.